spec/resources/numismatic_provenances/numismatic_provenance_decorator_spec.rb

```diff
@@ -14,21 +14,9 @@
     end
   end
 
-  describe "#firm" do
-    it "renders the provenance firm" do
-      expect(decorator.firm).to eq("name, city")
-    end
-  end
-
-  describe "#person" do
-    it "renders the provenance person" do
-      expect(decorator.person).to eq("name1 name2")
-    end
-  end
-
   describe "#title" do
     it "renders the provenance title" do
-      expect(decorator.title).to eq("name, city; name1 name2; 12/04/1999; note")
+      expect(decorator.title).to eq("firm name, firm city; name1 name2; 12/04/1999; note")
```

Comment:
This seems like it might be needed to restore breaking builds on the `master` branch, I have ported it to https://github.com/pulibrary/figgy/tree/hotfix-numismatic-prov-decorator-spec